src/sentry/web/helpers.py

```diff
@@ -4,96 +4,22 @@
 from typing import Any, Mapping
 
 import pytz
-from django.conf import settings
-from django.contrib.auth.models import AnonymousUser
 from django.http import HttpRequest, HttpResponse
 from django.template import loader
 from django.utils import timezone
 
-from sentry.auth import access
-from sentry.models import Team
 from sentry.utils.auth import get_login_url  # NOQA: backwards compatibility
-from sentry.utils.settings import is_self_hosted
 
 logger = logging.getLogger("sentry")
 
 
-def get_default_context(
```

Comment:
YUM @stefanfoulis none of this is referenced in templates anymore!  Instead, most of this gets stuffed into the client config for react pages only.